composer.json

```diff
@@ -16,9 +16,9 @@
         "php": ">=7.1",
         "symfony/symfony": "^3.4",
         "jms/translation-bundle": "^1.3.2",
-        "ezsystems/ezpublish-kernel": "^7.1",
-        "ezsystems/repository-forms": "^2.1",
-        "ezsystems/ezplatform-admin-ui-modules": "^1.1",
+        "ezsystems/ezpublish-kernel": "^7.1@beta",
```

Comment:
This could also be `@dev` if this package really depend on 7.1 now, same for changes below.

`@dev` is what we would have used if this was done as part of PR that introduced 7.1 dependency.